gophpserialize.go

```diff
@@ -224,7 +230,7 @@ func (s *Serializer) move() {
 func Unmarshal(data []byte) (rv interface{}, err error) {
 	defer func() {
 		if r := recover(); r != nil {
-			err = fmt.Errorf("Cant Unmarshal data %v, error: %v", data, r)
+			err = fmt.Errorf("cant Unmarshal data, error: %v", r)
```

Comment:
Not quite happy with this error message as well: 
- `cant` either `can't` or better `cannot`
- `Unmarshal` -> `unmarshal` since it's a verb

Reply:
Good call! Addressed in https://github.com/zalora/gophpserialize/pull/2/commits/1d6f4ebbb3ba6b22b19d42569630c94e006b2c77